set.mm

```diff
@@ -58687,6 +58687,130 @@ singleton of the first member (with no sethood assumptions on ` B ` ).
     ( c0 cpred ccnv csn cima cin df-pred 0in eqtri ) CABDCAEBFGZHCCABILJK $.
 
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Founded partial induction
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+
+  ${
+    $d A x y z w $.  $d R x y z w $.  $d B x y z w $.
+    $( Every (possibly proper) subclass of a class ` A ` with a founded,
+       partial-ordering, set-like relation ` R ` has a minimal element.  The
```

Comment:
A more standard terminology is that R is a "well-founded set-like partial order". More generally:
- "founded" --> "well-founded",
- we use more common term for partial orders is that one, "partial order", so let's stick to it for consistency.

Also:
- "Every" --> "Every nonempty"
- "avoiding infinity" --> "avoiding the axiom of infinity"

There are other occurrences below.